surveys/templates/location_list.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block extracss %}
+  <link rel="stylesheet" href="{% static 'css/jquery.dataTables.min.css' %}">
+{% endblock %}
+
+{% block extrajs %}
+  <script type="text/javascript" src="{% static 'js/jquery.dataTables.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/dataTables.bootstrap4.min.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+
+<div class="card">
+  <div class="card-body">
+    <h2 class="card-title">Locations</h2>
+    <small class="form-text text-muted">Text describing what a location is.</small>
+    <br />
+
+    <table id="locations-list" class="table table-hover table-striped table-responsive-sm card-text">
+    <thead>
+      <tr>
+        <th scope="col">Primary Name</th>
+        <th scope="col">Geometry Type</th>
+        <th scope="col">Subdivision</th>
+        <th scope="col">Character</th>
+        <th scope="col">Delete</th>
```

Comment:
Since we're centering the Delete icon, we should probably center this column name as well:

```suggestion
        <th class="text-center" scope="col">Delete</th>
```

Reply:
👍🏻I'll get this for all the lists